src/App/App.tsx

```diff
@@ -63,21 +70,23 @@ class Main extends React.Component<{}, {}> {
       document.title = getPageMetadata(window.location.pathname).title;
     }
     return (
-      <Switch>
-        <Route exact={true} path={BASEPATH} component={withWrappers(Home)} />
-        <Route exact={true} path={`${BASEPATH}/about`} component={withWrappers(About)} />
-        <Route exact={true} path={`${BASEPATH}/g/:gameCode`} component={withWrappers(GameInfo)} />
-        <Route exact={true} path={`${BASEPATH}/g/:gameCode/:mode`} component={withWrappers(Game)} />
-        <Route exact={true} path={`${BASEPATH}/g/:gameCode/:mode/:aiLevel`} component={withWrappers(Game)} />
-        <Route
-          exact={true}
-          path={`${BASEPATH}/g/:gameCode/:mode/:matchCode/:playerID`}
-          component={withWrappers(Game)}
-        />
-        <Route path={`${BASEPATH}/room/new/:gameCode/:numPlayers`} exact={true} component={withWrappers(NewRoom)} />
-        <Route path={`${BASEPATH}/room/:gameCode/:roomID`} exact={true} component={withWrappers(Room)} />
-        <Route exact={true} component={withWrappers(getMessagePage('error', 'Not Found'))} />
-      </Switch>
+      <MuiThemeProvider theme={theme}>
```

Comment:
What's the purpose or this change? Why moving the muithemeprovider here without deleting anywhere else?

Reply:
https://github.com/mui-org/material-ui/issues/13175#issuecomment-428480755

I was trying to get rid of some of the warnings, and you need to have a theme to use Typography v2.  It didn't eliminate all of the warnings, though.

Reply:
But we should have a theme everywhere we test/run the app, no? If we don't, can't we fix exactly where we don't instead of trying to apply this everywhere (and possibly twice?)

Reply:
Not sure what you mean by this.  AFAIK we don't have any other themes across the app?